pkg/registry/core/node/strategy.go

```diff
@@ -58,6 +59,29 @@ func (nodeStrategy) NamespaceScoped() bool {
 	return false
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (nodeStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	fields := map[fieldpath.APIVersion]*fieldpath.Set{
+		"v1": fieldpath.NewSet(
+			fieldpath.MakePathOrDie("status"),
+		),
+	}
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) {
+		fields["v1"].Insert(fieldpath.MakePathOrDie("spec", "configSource"))
+		fields["v1"].Insert(fieldpath.MakePathOrDie("status", "config"))
+	}
+
+	if !utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) {
+		// TODO: probably not correct, as reset only resets to the first entry
```

Comment:
What should we do with fields like this?
Just live with it not being perfect for now and ignore/not ignore?